config/jobs/kubernetes/sig-node/containerd.yaml

```diff
@@ -17,7 +17,7 @@ presets:
   - name: NON_MASQUERADE_CIDR
     value: 0.0.0.0/0
   - name: KUBELET_TEST_ARGS
-    value: --runtime-cgroups=/system.slice/containerd.service
+    value: "--runtime-cgroups=/system.slice/containerd.service --cgroup-driver=systemd"
```

Comment:
This assumes that all the jobs with `preset-e2e-containerd` should use systemd cgroup driver, is it accurate?

Reply:
We should not need this -- for cluster e2e tests, the GCE node bootstrap scripts already configure `--cgroup-driver=systemd` on cgroupv2 OS images (see https://github.com/kubernetes/kubernetes/blob/master/cluster/gce/gci/configure-helper.sh#L1630-L1633). So we only need to configure node e2e tests, not cluster e2e tests.

Reply:
Should we have a separate label preset for those jobs then? 

Just thinking out loud. . We can have a new preset for tests that use cos image and `preset-e2e-containerd` will be as is without any changes.  Will it impact any other tests? Or that would be a huge change to do.

Reply:
> Should we have a separate label preset for those jobs then?

That's one option, we don't have too many cluster e2e tests in `config/jobs/kubernetes/sig-node/containerd.yaml`, so I think we can just also just update each them or setup a present and annotate with preset, up to you.

The important part is we only update the ones for systemd cgroup driver on the tests which are using cgroupv2 images.

Reply:
Except these 4 jobs, all of them have either systemd driver set or is using an older cos image
- [ci-cos-containerd-node-e2e](https://github.com/kubernetes/test-infra/blob/f89c3d9701ef9fb70caeadebab2865069651863e/config/jobs/kubernetes/sig-node/containerd.yaml#L893)
- [ci-cos-containerd-node-e2e-features](https://github.com/kubernetes/test-infra/blob/f89c3d9701ef9fb70caeadebab2865069651863e/config/jobs/kubernetes/sig-node/containerd.yaml#L929)
- [ci-kubernetes-node-kubelet-lock-contention](https://github.com/kubernetes/test-infra/blob/master/config/jobs/kubernetes/sig-node/containerd.yaml#L1338)
- [ci-kubernetes-node-kubelet-credential-provider](https://github.com/kubernetes/test-infra/blob/f89c3d9701ef9fb70caeadebab2865069651863e/config/jobs/kubernetes/sig-node/containerd.yaml#L1367)

The above jobs have multiple images (ubuntu and cos stable) specified in the image config. So I am not sure how to update the `--cgroup-driver` value for these jobs.

Reply:
I think the general rule we can follow is:

* If the job is pointing to an image config file that is using something older than COS M97 (or != `cos-stable`), then let's keep it as is for now
* Otherwise, let's update the image config to `cos-stable` and `ubuntu 22.04` and also update cgroup driver to systemd.